### DISCUSSION TITLE
Non riesco più ad entrare con http://homeassistant.local:8123/ dopo aver istallato DUCK DNS

### DISCUSSION THREAD

**Discussion by francescobuonfigliodomotica** [indomus/forum#436] [HUB: Home Assistant]
Ciao a tutti. Col supporto della vostra guida per l’istallazione di DUCK DNS in HASSIO sono riuscito finalmente a portare a termine l’istallazione e anche l’aggiornamento delle certificazioni all’interno delle integrazioni.
I quesiti che vi pongo sono questi:
•	È corretto che io non possa più entrare in HA tramite "http://homeassistant.local:8123/"? Se non è corretto, quale log devo guardare per capire quale errore ho fatto? In quello di DUCK DNS è tutto regolare;
•	Quando entro con "https://192.168.178.31:8123/" , mi trovo https barrato e rosso nella casella del browser e la scritta “non sicuro” a fianco. Pensavo di aver risolto con le certificazioni, ma chiaramente sono in errore. Cosa potrei fare per non avere più la scritta "non sicuro"?
Grazie in anticipo a chi può aiutarmi. Francesco

**Comment by maxcanna** [comment]
Quel che hai scritto è tutto assolutamente normale.

- avendo abilitato https non riesci più ad accedere in http perché ha ascolta in https o in http non entrambi nello stesso momento
- accendendo con indirizzo IP il tuo browser vede che il certificato presentato da ha non corrisponde al sito che vuoi visitare e ti avvisa

**Reply by francescobuonfigliodomotica** [reply]
Ok, tutto chiaro.
Grazie mille maxcanna.
